www/roster/main.rb

```diff
@@ -167,6 +167,14 @@
   _html :podlings
 end
 
+# gives a list of podlings that would report in a given month based on current status
+get '/podlings/reporting/:month.json' do |month|
```

Comment:
Understood.  And actually, that's why I prefer using sinatra for what this endpoint is doing.  I noticed that the data was loaded once and cached for a period of time, so unlike using cron to write a file, this wouldn't end up being extremely out of date, but shortly out of date if no one is using the ASF::Podling class.

I am the kind of person who jumps deep end first.  I do expect this to go in baby steps, but maybe also refactoring is required (e.g. /roster/ppmc/ shouldn't be protected).  But we can talk about that part later.